traja/models/train.py

```diff
@@ -315,21 +328,21 @@ def fit(
                                 regressor_out, parameters
                             )
 
-                test_loss_forecasting /= len(test_loader.dataset)
+                test_loss_forecasting /= len(data_loader_to_evaluate.dataset)
                 print(
-                    f"====> Mean test set forecasting loss: {test_loss_forecasting:.4f}"
+                    f"====> Mean {current_set} set forecasting loss: {test_loss_forecasting:.4f}"
```

Comment:
You did not update the name of `test_loss_forecasting` to reflect its new usage.

Reply:
Done.